test/transactions/transactionBytes.js

```diff
@@ -15,50 +15,250 @@
 import { getTransactionBytes } from '../../src/transactions/transactionBytes';
 
 describe('#getTransactionBytes', () => {
-	let bytes = null;
+	const defaultRecipient = '58191285901858109L';
+	const defaultSenderPublicKey = '5d036a858ce89f844491762eb89e2bfbd50a4a0a0da658e4b2628b25b117ae09';
+	const defaultSenderSecondPublicKey = '0401c8ac9f29ded9e1e4d5b6b43051cb25b22f27c7b7b35092161e851946f82f';
+	const defaultAmount = 1000;
+	const defaultNoAmount = 0;
+	const defaultTimestamp = 141738;
+	const defaultTransactionId = '13987348420913138422';
+	const defaultSignature = '618a54975212ead93df8c881655c625544bce8ed7ccdfe6f08a42eecfb1adebd051307be5014bb051617baf7815d50f62129e70918190361e5d4dd4796541b0a';
+	const defaultSecondSignature = 'b00c4ad1988bca245d74435660a278bfe6bf2f5efa8bda96d927fabf8b4f6fcfdcb2953f6abacaa119d6880987a55dea0e6354bc8366052b45fa23145522020f';
+	const defaultAppId = '1234213';
 
-	it('should be ok', () => {
-		(getTransactionBytes).should.be.ok();
+	describe('send transaction, type 0', () => {
+		let defaultTransaction;
+
+		beforeEach(() => {
+			defaultTransaction = {
+				type: 0,
+				amount: defaultAmount,
+				recipientId: defaultRecipient,
```

Comment:
`fee`?

Reply:
Also https://docs.lisk.io/docs/the-lisk-protocol-transactions lists `senderId`, but not sure if that's still valid.